xmeffects.js

```diff
@@ -56,7 +56,7 @@ function eff_t1_3(ch) {  // portamento
 
 function eff_t0_4(ch, data) {  // vibrato
   if (data & 0x0f) {
-    ch.vibratodepth = data & 0x0f;
+    ch.vibratodepth = (data & 0x0f) * 2;
```

Comment:
[Go here](https://source.openmpt.org/browse/openmpt/trunk/OpenMPT/soundlib/Snd_fx.cpp) and search for `CSoundFile::Vibrato`.
OpenMPT multiplies this with 4 which was too much in this case, 2 made it sound just like it did when testing in FT2.